docs/development.md

```diff
@@ -46,25 +46,96 @@ Edit docker-compose.yml configuration file to change the default ports.
  ```
 ### Testing
 
-TODO: add
-
+We strive to have a full test coverage in Presidio, and expect every pull request to
+ include tests. 
```

Comment:
nit: the document (ex. this paragraph and the next - and also earlier in the doc) seems to have some odd spacing - new lines have an increasing amount of whitespace. While this doesn't seem to have an effect on the end-result it makes your mind wander while reading in markdown. So just commenting in case you want to clean it up.

Reply:
Also noticed line 31 (outside of this PR) successfuly -> successfully, since you are working in the file.

Reply:
Fixed both 👍